### PR TITLE
Add vol pricing to prod card

### PR DESCRIPTION
**Pull request by sofiamatulis**
### PR Summary: 

Add a note to products that contain vol pricing (prod card)

### Why are these changes introduced?

[Ticket](https://github.com/orgs/Shopify/projects/6397/views/5?pane=issue&itemId=30826314)

### Testing steps/scenarios

- [ ] Product with image
- [ ] Product without image
- [ ] Product with vol pricing
- [ ] Product without vol pricing

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/140178817046/editor)

### CODE REVIEW

File: snippets/card-product.liquid [hunk 1]
```diff
@@ -200,6 +200,11 @@
             {%- endif -%}
 
             {% render 'price', product: card_product, price_class: '' %}
+            {%- if card_product.quantity_price_breaks_configured? -%}
+              <div class="card__information--volume-pricing-note">
+                <span class="caption">{{ 'products.product.volume_pricing.note' | t }}</span>
```
Comment (eugenekasimov) [inline snippets/card-product.liquid:205]: this locales was added in different PR, is it right?

**Review by eugenekasimov — APPROVED** [review]

File: snippets/card-product.liquid (hunk 1 above)
Comment (sofiamatulis) [reply snippets/card-product.liquid:205]: Yes I branched off of that :)

**Review by scottgmeadows — APPROVED** [review]

File: snippets/card-product.liquid [hunk 2]
```diff
@@ -200,6 +200,11 @@
             {%- endif -%}
 
             {% render 'price', product: card_product, price_class: '' %}
+            {%- if card_product.quantity_price_breaks_configured? -%}
+              <div class="card__information--volume-pricing-note">
```
Comment (KaichenWang) [inline snippets/card-product.liquid:204]: Regarding the selector naming, why is this selector a modifier of `card__information`? What is the relationship between this element and the other `card__information` elements?

**Review by KaichenWang — APPROVED** [review]
Looks good!

One UX question: we may want to reduce the line-height?
Screenshot 2023-06-29 at 9 41 34 AM

File: snippets/card-product.liquid (hunk 2 above)
Comment (KaichenWang) [reply snippets/card-product.liquid:204]: If we're using BEM convention here, a modifier (`card__information--volume-pricing-note`) should not be applied to a child of `card__information`

**Pull request merged**